tests/test_sanity.py

```diff
@@ -889,7 +889,10 @@ def test_vanilla(self):
 
     def make_fake(report):
       f = open(CONFIG_FILE, 'a')
-      f.write('LLVM_ROOT = "' + path_from_root('tests', 'fake', 'bin') + '"\n')
```

Comment:
lgtm, thanks.

However, I am puzzled about the line with LLVM_ROOT, that shows up in the diff as being changed, but I don't see anything changed. Perhaps tabs/spaces or something else not visible on github?

Reply:
Yes, it does look like an extra space, I see locally. Pushed fix.